src/ components/EateryCard.js

```diff
@@ -0,0 +1,244 @@
+import React, { useState } from "react";
+import {
+  Card,
+  CardHeader,
+  Typography,
+  styled,
+  Grid,
+  Button,
+  Accordion,
+  AccordionSummary,
+  AccordionDetails,
+  CardContent,
+  CardActions,
+  Avatar,
+  Dialog,
+} from "@mui/material";
+import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
+
+const StyledCard = styled(Card)({
```

Comment:
I like the use of styled components, but if you're going to use them then your CSS is now code and you probably want to use some constants somewhere to get rid of all the duplication in the colors, font families, etc.